### PR TITLE
Sorted post with created date

### LINKED ISSUE #19: User Feed
### AC
- Sort by Created Date (Trending & Recent)

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Changes
- Sort with the latest, oldest, and trending
- Trending posts on basis of likes count

### Preview
[URL](https://deploy-preview-20--always-social.netlify.app/)

### CODE REVIEW

File: src/features/Posts/utils.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  let newDate = new Date(date);
+  newDate = String(newDate).split(" ");
+  return `${newDate[1]} ${newDate[2]}, ${newDate[3]}`;
+};
+
+/**
+ * @description Check if user liked the post
+ * @param {object} user
+ * @param {Array} likedBy
+ * @returns {Boolean} user liked the posts or not
```
Comment (sri-rishi) [inline src/features/Posts/utils.js:16]: Appreciation: like the way of describing functionality

**Review by sri-rishi — APPROVED** [review]
Code is understandable and clean, feel free to merge

**Review by simran61 — APPROVED** [review]
Looks good
Clean folder structure

**Review by yashraj2312 — APPROVED** [review]
Looks good to merge.

**Pull request merged**